File: examples/ModuleTests/Server/GameLoop.lua
local MathUtils = require(game:GetService("ReplicatedStorage").Utils.MathUtils)
local GameLoop = {}
GameLoop.__index = GameLoop
function GameLoop.new()
    local self = setmetatable({}, GameLoop)
    local result = MathUtils.Add(5, 10)
    print("MathUtils result: " .. result)
    return self
end
return GameLoop

File: examples/ModuleTests/Server/GameLoop.cs
using System;
using Shared.Utils;
using Roblox;
using Roblox.Services;

namespace Server.Game;

public class GameLoop
{
    public GameLoop()
    {
        int result = MathUtils.Add(5, 10);
        Console.WriteLine("MathUtils result: " + result);
    }
}
